backend/typescript/services/implementations/EntityService.ts

```diff
@@ -0,0 +1,51 @@
+import { errorMonitor } from "node:events";
+import { MEntity, Entity } from "../../models/entity.mgmodel";
+import { IEntityService, EntityRequestDTO } from "../interfaces/IEntityService";
+
+
+export class EntityService implements IEntityService {
+
+    async getEntity(id: string): Promise<Entity> {
+        let entity: Entity | null;
+        entity = await MEntity.findById(id);
+
+        if (!entity) {
+            throw new Error(`Entity id ${id} not found.`);
+        }
+        return entity;
+    }
+
+    async getEntities(): Promise<Entity[]> {
+        return await MEntity.find();
+    }
+
+    async createEntity(entity: Entity): Promise<Entity> {
+        try {
+            const newEntity: Entity | null = await MEntity.create(entity);
+            return newEntity;
+        }
+        catch (e) {
+            throw Error;
```

Comment:
We should be throwing the caught error `e` (I know the catch block is useless right now, but wait for the logger!)

Same comment for the update and delete methods.